Containerfile

```diff
@@ -37,18 +42,6 @@ LABEL \
 # Switch workdir
 WORKDIR /opt/apm-perf
 
-# Add minimal stuff
-RUN \
-  apt-get update > /dev/null \
-  && apt-get upgrade -y \
-  && apt-get install -y --no-install-recommends \
-    "apt-utils=*" \
-    "ca-certificates=*" \
-    "curl=*" \
-    "gnupg=*" \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
```

Comment:
Removing this will break the tilt/AWS setup that relies on `curl` to generate the API key.

Reply:
ugh :)

Thanks for catching that, didn't realise/forgot we were doing this.